proto/interchain_security/ccv/v1/ccv.proto

```diff
@@ -38,3 +39,14 @@ message UnbondingDelegationEntry {
   // child chains that are still unbonding
   repeated string unbonding_child_chains = 3;
 }
+
+// This packet is sent from the consumer chain to the provider chain.
+// The acknowledgement will be sent asynchrounously when the jailing 
+// will be started on the provider chain.
+message ValidatorDowntimePacketData {
```

Comment:
Are you creating separate packet data type for each type of evidence?

My understanding of v1, was that we just send back a generic evidence packet with validator and slash fraction

Reply:
I think that Simon is generalizing this in his current work